### ISSUE TITLE
Support for markup in drag element

### ISSUE THREAD

**Issue by ThomasKlessen** [Vivify-Ideas/vue-draggable#7]
```
<div v-drag-and-drop:options="options">
  <ul>
    <li><label>Item 1</label></li>
    <li><label>Item 2</label></li>
    <li><label>Item 3</label></li>
  </ul>
  <ul>
      <li><label>Item 4</label></li>
      <li><label>Item 5</label></li>
      <li><label>Item 6</label></li>
  </ul>
</div>
```

We want to display more information within the drag element (the example is simple; In reality there will be a img and some additional spans/labels).
Everything is working fine but i get a `DOMException` in `vue-draggable.js@662` with the message: `Failed to execute 'insertBefore' on 'Node': The node before which the new node is to be inserted is not a child of this node.`

So i wrapped the code in try catch and everything meets my expectations, but i think this should not be the general solution. 

I would love to help with this Feature Request but need some input how it should be done?

I could imagine something like going upword in the dom until i find a li ?

**Comment by nikolasp** [comment]
@ThomasKlessen Thanks for opening issue... You are right, try-catch isn't good solution for this. This must be handled by library itself. I will try to fix this as soon as possible and publish new version to npm. I will add some tests later, so this situation will be covered.

**Comment by nikolasp** [comment]
@ThomasKlessen This issue should be fixed with v1.0.6. 
Please let us know whether you are still facing the problem. 😄

**Comment by nikolasp** [comment]
solved issue 🚀
